README.md

```diff
@@ -1,25 +1,61 @@
-<p align="left">
+<p align="center">
     <a href="https://datafold.com/"><img alt="Datafold" src="https://user-images.githubusercontent.com/1799931/196497110-d3de1113-a97f-4322-b531-026d859b867a.png" width="30%" /></a>
 </p>
 
-<h1 align="left">
-data-diff: compare datasets fast, within or across SQL databases
-</h1>
+<h2 align="center">
+data-diff: Compare datasets fast, within or across SQL databases
 
+![data-diff-logo](docs/data-diff-logo.png)
+</h2>
 <br>
 
+# Use Cases
+
+## Data Migration & Replication Testing
+Compare source to target and check for discrepancies when moving data between systems:
+- Migrating to a new data warehouse (e.g., Oracle > Snowflake)
+- Converting SQL to a new transformation framework (e.g., stored procedures > dbt)
+- Continuously replicating data from an OLTP DB to OLAP DWH (e.g., MySQL > Redshift)
+
+
+## Data Development Testing
+Test SQL code and preview changes by comparing development/staging environment data to production:
+1. Make a change to some SQL code
+2. Run the SQL code to create a new dataset
+3. Compare the dataset with its production version or another iteration
+
+  <p align="left">
+  <img alt="dbt" src="https://seeklogo.com/images/D/dbt-logo-E4B0ED72A2-seeklogo.com.png" width="10%" />
+  </p>
+  
+<details>
+<summary> Integrates with dbt Core and dbt Cloud to seamlessly compare local development to production datasets. </summary>
```

Comment:
This is something that I was going to update, and already discussed with Gleb needs to be updated (he agreed):  

`data-diff` does not integrate with dbt Cloud. I would say that _Datafold_ integrates with dbt Cloud, but it's misleading to say that `data-diff` integrates with dbt Cloud. Because in order to use it with dbt Cloud, well, you can't -- you have to install dbt Core.

This came up recently because I got a question about the statement ("how does data-diff integrate with dbt Cloud?") pointing to this line in the readme. I had to say, "you're right--it doesn't. I'll update the readme.

Reply:
Here's a suggested update:

> data-diff integrates with dbt Core to seamlessly compare local development to production datasets. dbt Cloud users should check out [Datafold's out-of-the-box development testing integration with dbt Cloud](https://www.datafold.com/data-deployment-testing).